litellm/__init__.py

```diff
@@ -366,6 +371,11 @@ def identify(event_details):
     "meta-llama/Llama-2-70b-chat",
 ]  # these have been tested on extensively. But by default all text2text-generation and text-generation models are supported by liteLLM. - https://docs.litellm.ai/docs/providers
 
+empower_models = [
```

Comment:
@liuyl this would override the models pulled from the model_prices_and_context_window.json